pylint/checkers/base/name_checker/checker.py

```diff
@@ -150,6 +158,12 @@ class NameChecker(_BasicChecker):
             "do not require a suffix. The message is also emitted when invariant "
             "variables do have a suffix.",
         ),
+        "C0106": (
```

Comment:
Could you use another number? The `0106` is already emitted as `E0106`. We have a check for reusing numbers, but for these lower numbers there are already conflicting numbers. (There is also `W0106`) Therefore, it is turned off for `0106`.

Reply:
will do, FWIW the checks immediately above this line (C0103-C0105) also clash with E and W error codes